src/sentry/api/endpoints/project_details.py

```diff
@@ -142,7 +142,18 @@ def validate(self, data):
         try:
             config_str = json.dumps(data)
             validate_sampling_configuration(config_str)
-            self.validate_uniform_sampling_rule(data.get("rules", []))
+
+            # If the feature flag 'organizations:dynamic-sampling-demo' is enabled, we skip the uniform rule validation.
+            # This is useful for product demos, as the user will be able to delete uniform rules.
+            if (
+                features.has(
+                    "organizations:dynamic-sampling-demo",
+                    self.context.get("project", {}).organization,
+                    actor=self.context.get("request", {}).user,
```

Comment:
same here.

Reply:
in out code we are usually writing `context["request"].user` but I thought that `get` was a better pattern ... This data is always going to be available

Reply:
```
self.context["request"].user,
```

Reply:
which of the two options is better?

Reply:
you have only one option, also it's not about better or not. UseCase for `get` is slightly different.

Reply:
I would stay with `[]` for consistency and because it will throw a `KeyError` whereas `get` will return `None`.